// my_service.proto

syntax = "proto3";

package myservice;

// 请求消息
message HelloRequest {
  string name = 1;
  int32 index = 2;
}

// 响应消息
message HelloReply {
  string message = 1;
  int32 index = 2;
}

// 定义 MyService 服务
service MyService {
  rpc SayHello (HelloRequest) returns (HelloReply);
}
